parent/pom.xml

```diff
@@ -746,6 +746,8 @@
           <configuration>
             <comparisonVersion>${backwards.compat.version}</comparisonVersion>
             <ignoredDifferencesFile>${ignored.changes.file}</ignoredDifferencesFile>
+            <!-- TODO(menski): Enable again after 1.0 release -->
+            <skip>true</skip>
```

Comment:
Can we create a follow up issue so we don't forget?

Reply:
Created issue https://github.com/zeebe-io/zeebe/issues/6499